### ISSUE TITLE
As this year's event completes, show the date for next year's #GAAD

### ISSUE THREAD

**Issue by nfreear** [nfreear/gaad-widget#5]
Suggested by: @srinivasuchakravarthula on the wishlist, #2.

* Comment: https://github.com/nfreear/gaad-widget/issues/2#issuecomment-302639338

**Comment by nfreear** [comment]
Hi @srinivasuchakravarthula,

I've released [2.0-beta] incorporating your suggestion. Enjoy!

Thanks for the feedback & support ;)


Nick

---
```html
<div id="id-gaad"></div>

<script src="https://cdn.rawgit.com/nfreear/gaad-widget/2.0-beta/build/GAAD.widget.js"></script>
```

[2.0-beta]: https://github.com/nfreear/gaad-widget#v2x

**Comment by srinivasuchakravarthula** [comment]
That's awesome. Thank you. 

Regards,
Srinivasu Chakravarthula
+91-9900810881
Sent on my phone. Excuse typos, if any.
